lib/parser/tensorflow/tf_parser.cc

```diff
@@ -76,6 +81,64 @@ void TFParser::Init(BasicBlock* bb, Function* function,
   opts_ = opts;
 }
 
+Status TFParser::SetAttributes(tensorflow::NodeDef* cur_node) {
```

Comment:
Change to "SetIPUAttributes()" ?

Reply:
Can we move this code to a separate file? We can have a TFParser subclass like 
class IPUConverter : TFParser() {
...
}

Reply:
tf_parser class depends on tf_convert.h.inc, IPUConvert cannot be decoupled cleanly, need further refactor